lms/static/scripts/frontend_apps/components/LMSFilePicker.tsx

```diff
@@ -398,16 +416,16 @@ export default function LMSFilePicker({
             <span>Unable to authorize file access.</span>
           ) : (
             <span>
-              To select a file, you must authorize Hypothesis to access your
-              files.
+              To select a {documentType}, you must authorize Hypothesis to
+              access your {documentType}s.
```

Comment:
I have decided to be pragmatic, and simply use the `documentType` value (page or file) and pluralized it by appending an `s`, which is correct for those two possible values.

If this evolves into supporting more types or requiring completely different messages for every type, we will need some helper functions and adjust this.

Reply:
We'd also need to overhaul this to support translation.